components/Comment/CommentEditorToolbar.tsx

```diff
@@ -0,0 +1,76 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFontCase } from "@fortawesome/pro-solid-svg-icons";
+import { faChevronUp } from "@fortawesome/pro-regular-svg-icons";
+import { useEffect, useRef, useState } from "react";
+import isOutsideClick from "~/config/utils/isOutsideClick";
+
+type Args = {
+  editorId: string;
+};
+
+const CommentEditorToolbar = ({ editorId }: Args) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const toolbarRef = useRef(null);
+
+  useEffect(() => {
+    const _handleClick = (e) => {
+      const _isOutsideClick = isOutsideClick({
+        el: toolbarRef.current,
+        clickedEl: e.target,
+        exclude: [".ql-full-editor-visible"],
+      });
+
+      if (_isOutsideClick) {
+        setIsOpen(false);
+      }
+    };
+
+    document.addEventListener("click", _handleClick);
+
+    return () => {
+      document.removeEventListener("click", _handleClick);
+    };
+  }, []);
+
+  return (
+    <div id={editorId} className="ql-toolbar" ref={toolbarRef}>
+      <span className="ql-formats">
```

Comment:
`className` need to be as such in order for quill to work properly

Reply:
can you leave this comment within code.